### DISCUSSION TITLE
Is it possible to View an outq or work with spool files? I'm kind of new to this

### DISCUSSION THREAD

**Discussion by jhullsdc** [codefori/vscode-ibmi#1797] [Q&A] [answered]
Hey all,

I'm new to VS Code for IBM. Is there a way I can view my outq or all my spool files? Some programs will make spool files and I would hate to go back to the green screen every time I wanted to look at something.

Thanks!

**Comment by sebjulliand** [comment, marked as answer]
Hello @jhullsdc ,
it is not possible as of now. There is, however, an extension in the making that will eventually add support for browsings spooled files and OUTQ maybe: https://github.com/codefori/vscode-ibmi-fs

It's not been released yet but we're working on it!

**Reply by jhullsdc** [reply]
Thanks for the response! I appreciate it.

A few of the guys here aren't wanting to transition until spools are available, but to me it's so good, why wait? :)

**Reply by sebjulliand** [reply]
That's the spirit! 😁

**Comment by thomprl** [comment]
This will be a cool addition.
